docs/specs/servers/notify/notify-server-api.md

```diff
@@ -2,183 +2,244 @@
 
 ## Authentication
 
-All endpoints expect an `Authorization` header in the form `Authorization: Bearer <project_secret>` using the project secret associated with a project ID. The secret used should be the one that was generated automatically when configuring notify - with the name `notify_api_secret`.
+All endpoints expect an `Authorization` header in the form `Authorization: Bearer <notify_api_secret>` using the notify_api_secret associated with a project ID. The secret used should be the one that was generated automatically when configuring notify - with the name `notify_api_secret`.
 
-## Notify
+## Send Notifications
 
-Send a notification to a set of accounts.
+Send 1 or more notifications to accounts. Notifications handled itempotently are deduplicated by `notificationId`.
 
-`POST /notify`
-
-Body:
+`POST /v1/<project-id>/notify`
 
 ```typescript
-{
-  "notification": {
-    "title": string,
-    "body": string,
-    "icon": string,
-    "url": string,
-    "type": string,
+type Body = [{
+  // Idempotency key and ID to track status
+  notificationId?: string | null,
+  notification: {
+    // The notification type ID found in [Notify Config](../../clients/notify/notify-config.md)
+    type: string,
+    title: string,
+    body: string,
+    icon?: string | null,
+    url?: string | null,
```

Comment:
Shouldn't be nullable

Reply:
@Elyniss it should because this is the Notify API endpoint. `null` will be translated to `""` by the Notify Server